src/main/java/com/bytebreak/animagic/texture/AnimagicTextureData.java

```diff
@@ -0,0 +1,9 @@
+package com.bytebreak.animagic.texture;
+
+/**
+ * Created by Admin on 11/29/2015.
+ */
+public class AnimagicTextureData {
```

Comment:
We should maybe just make this immutable instead of having publicly accessible variables.  We can just put the metadata in the constructor.